staging/src/k8s.io/apimachinery/pkg/watch/until.go

```diff
@@ -45,7 +45,7 @@ func Until(timeout time.Duration, watcher Interface, conditions ...ConditionFunc
 		after = time.After(timeout)
 	} else {
 		ch := make(chan time.Time)
-		defer close(ch)
+		close(ch)
```

Comment:
this doesn't look right to me.  why create a chan if in the next line we close it?